Library/LocationTracker.swift

```diff
@@ -57,9 +60,11 @@ extension LocationTracker: CLLocationManagerDelegate {
         geocoder.reverseGeocodeLocation(locations.first!) { placemarks, error in
             if let error = error {
                 self.delegate?.locationTracker(self, didFailWith: error)
-            } else if let placemarks = placemarks {
-                let placemark = placemarks.first!
-                self.delegate?.locationTracker(self, didFindLocation: placemark)
+            } else if let placemarks = placemarks, let placemark = placemarks.first {
+                if self.placemark?.country != placemark.country && self.placemark?.locality != placemark.locality {
```

Comment:
Can't compare them, because they have floating point accuracy for the coordinates. So they don't usually match. I'm fine with it as it is for now. There's a major refactoring on #107, I can redo that there if you don't mind?

Reply:
But why do you only update twhen the country changes? Shouldnt it be or?

` if self.placemark?.country != placemark.country || self.placemark?.locality != placemark.locality {`

Reply:
🤦🏽‍♂️